lighthouse-core/gather/gather-runner.js

```diff
@@ -27,46 +27,34 @@ class GatherRunner {
     return driver.gotoURL('about:blank')
```

Comment:
I wanted to follow the overall sequencing here so I wrote it out. Could we add this in as a comment?


#### GR.run
1. instantiateGatherers
1. driver.connect()
1. GR.setupDriver
  1. startEmulation
  1. cleanAndDisableBrowserCaches
  1. forceUpdateServiceWorkers
1. forEach pass in the config:
  1. GR.beforePass
  	1. all gatherer's beforePass()
  1. GR.pass
    1. loadPage 
      1. navigate to about:blank
      1. beginTrace & beginNetworkCollect
      1. navigate to options.url (and wait for onload)
    1. all gatherer's pass()
  1. GR.afterPass
    1. endTrace, endNetworkCollect
    1. all gatherer's afterPass()
1. reloadForCleanStateIfNeeded
1. driver.disconnect()
1. collect all artifacts and return them